src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -0,0 +1,213 @@
+
+  /**
+   * Indicates whether the input is optional.
+   */
+  @Prop() public optional?: boolean;
+
+  /**
+   * Size variant, either l or m.
+   */
+  @Prop() public size?: InterfaceSbbFormFieldAttributes['size'] = 'l';
+
+  /** Whether to display the form field without a border. */
+  @Prop() public borderless = false;
+
+  /**
+   *
+   * It is used internally to get the native `disabled` attribute from `<input>`.
+   */
+  @State() private _disabled: boolean;
+
+  /**
+   *
+   * It is used internally to get the native `readonly` attribute from `<input>`.
+   */
+  @State() private _readonly: boolean;
+
+  /**
+   * It is used internally to get the `prefix` slot.
+   */
+  @State() private _prefixElements: Element[] = [];
+
+  /**
+   * It is used internally to get the `suffix` slot.
+   */
+  @State() private _suffixElements: Element[] = [];
+
+  /**
+   * It is used internally to get the `suffix` slot.
+   */
+  @State() private _errorElements: Element[] = [];
+
+  /** Whether the input inside the form field is invalid. */
+  @State() private _invalid = false;
+
+  private _originalInputAriaDescribedby?: string;
+
+  /**
+   * It is used internally to get the `input` slot.
+   */
+  private _input: HTMLInputElement | HTMLSelectElement | HTMLElement;
+
+  /**
+   * Listen the changes on `readonly` and `disabled` attributes of `<input>`.
+   */
+  private _formFieldAttributeObserver = new MutationObserver((mutations: MutationRecord[]) =>
+    this._onAttributesChange(mutations)
+  );
+
+  public disconnectedCallback(): void {
+    this._formFieldAttributeObserver.disconnect();
+  }
+
+  /**
+   * It is used internally to set the aria-describedby attribute for the slotted input referencing available <sbb-form-erro> instances.
+   */
+  private _onSlotErrorChange(event: Event): void {
+    this._errorElements = (event.target as HTMLSlotElement).assignedElements();
+    const value = this._errorElements.length
+      ? this._errorElements.map((e) => e.id).join(',')
+      : this._originalInputAriaDescribedby;
+    this._input.setAttribute('aria-describedby', value);
+  }
+
+  /**
+   * It is used internally to assign the attributes of `<input>` to `_id` and `_input` and to observe the native readonly and disabled attributes.
+   */
+  private _onSlotInputChange(event: Event): void {
+    this._input = (event.target as HTMLSlotElement).assignedElements()[0] as HTMLElement;
+    this._originalInputAriaDescribedby = this._input.getAttribute('aria-describedby');
+
+    this._readonly = this._input.hasAttribute('readonly');
+    this._disabled = this._input.hasAttribute('disabled');
+
+    this._formFieldAttributeObserver.observe(this._input, {
+      attributes: true,
+      attributeFilter: ['readonly', 'disabled', 'class'],
+    });
+
+    if (!this._input.id) {
+      this._input.id = `sbb-form-field-input-${nextId++}`;
+    }
+  }
+
+  /**
+   * @private
+   * It is used internally to set the focus in the `<input>`.
```

Comment:
```suggestion
   * It is used internally to set the focus to the input element.
```
can also be a select